From 90451f240d83cfb2797f22697e329a31cda40079 Mon Sep 17 00:00:00 2001
From: Martin Schwan <m.schwan@phytec.de>
Date: Mon, 15 Feb 2021 14:53:08 +0100
Subject: [PATCH 1/2] include: configs: Change A/B system partitions

Change the A/B system layout to the following scheme and adjust the
partition numbering accordingly:

    partition name  partition number
    --------------  ----------------
    boot0           1
    boot1           2
    config          3
    extended        4
        root0       5
        root1       6
        app0        7
        app1        8

Signed-off-by: Martin Schwan <m.schwan@phytec.de>
---
 include/configs/phycore_rauc_env.h | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/include/configs/phycore_rauc_env.h b/include/configs/phycore_rauc_env.h
index 48b15556ef..8e6fee95ac 100644
--- a/include/configs/phycore_rauc_env.h
+++ b/include/configs/phycore_rauc_env.h
@@ -35,11 +35,11 @@
 #define PHYCORE_RAUC_ENV_BOOTLOGIC \
 	"raucslot=system0\0" \
 	"raucbootpart=1\0" \
-	"raucrootpart=2\0" \
+	"raucrootpart=5\0" \
 	"raucbootpart0=1\0" \
-	"raucrootpart0=2\0" \
-	"raucbootpart1=3\0" \
-	"raucrootpart1=4\0" \
+	"raucrootpart0=5\0" \
+	"raucbootpart1=2\0" \
+	"raucrootpart1=6\0" \
 	"raucargs=setenv bootargs console=${console} " \
 		"root=/dev/mmcblk${raucdev}p${raucrootpart} " \
 		"rauc.slot=${raucslot} rootwait rw\0" \
-- 
2.30.1

